bin/stress-test/src/endpoints_metrics.rs

```diff
@@ -0,0 +1,22 @@
+use std::time::Duration;
+
+/// Prints a summary of the benchmark results.
```

Comment:
nit: We could explain that the results summary involves average time and p95 percentile